downstream/modules/eda/con-credentials-list-view.adoc

```diff
@@ -0,0 +1,12 @@
+[id="eda-credentials-list-view"]
+
+= Credentials list view
+
+On the *Credentials* page, you can view the list of created credentials that you have created along with the *Type* of credential.
+
+From the menu bar, you can search for credentials in the *Name* field. 
+
+You also have the following options in the menu bar:
+
+* Choose which columns are shown in the list view by clicking btn:[Manage columns].
+* Choose between a btn:[List view] or a btn:[Card view], by clicking the respective icons.
```

Comment:
IBM style guide recommends against using "respective"
See: "Do not use. Rewrite to avoid using this word. For example, write “Return each drive to its bay”, not “Return the drives to their respective bays”."